build-all.sh

```diff
@@ -50,6 +52,11 @@ if [[ ! "$TERMUX_ARCH" =~ ^(all|aarch64|arm|i686|x86_64)$ ]]; then
 	exit 1
 fi
 
+if [[ ! "$TERMUX_FORMAT" =~ ^(debian|pacman)$ ]]; then
+	echo "ERROR: Invalid format '$TERMUX_FORMAT'" 1>&2
+	exit 1
+fi
```

Comment:
Consider using a `case` for this instead of regex.
```suggestion
case "$TERMUX_FORMAT" in
	debian|pacman);;
	*) echo "ERROR: Invalid format '$TERMUX_FORMAT'" 1>&2; exit 1;;
esac
```